modules/op-constructing-pipelines-using-pipeline-builder.adoc

```diff
@@ -15,6 +15,13 @@ In the *Developer* perspective of the console, you can use the *+Add* -> *Pipeli
 * Reference these pipeline resources in each of the tasks in the pipeline as input and output resources.
 * If required, reference any additional parameters added to the pipeline in the task. The parameters for a task are prepopulated based on the specifications of the task.
 * Use the Operator-installed, reusable snippets and samples to create detailed pipelines.
+* Search and add tasks from your configured local Tekton Hub instance.
+
+[IMPORTANT]
+====
+In the developer perspective, you can create a customized pipeline using your own set of curated tasks. For allowing you to search, install, and upgrade your tasks directly from the developer console, your cluster administrator needs to install and deploy a local Tekton Hub instance and link that hub to the OpenShift Container Platform cluster. For more details, see _Using Tekton Hub with OpenShift Pipelines_ in the _Additional resources_ section.
```

Comment:
I think it'd read better if you said, `For you to search, install, and upgrade...your cluster administrator needs to install...` or `To search, install, and upgrade...your cluster administrator needs to install...`

From a minimalism principle, this also cuts down word count which is good :)

Reply:
I aggree with you @jc-berger. Incorporated your suggestion.